libsyclinterface/include/dpctl_sycl_kernel_interface.h

```diff
@@ -39,25 +39,124 @@ DPCTL_C_EXTERN_C_BEGIN
  */
 
 /*!
- * @brief Returns the number of arguments for the OpenCL kernel.
+ * @brief Returns the number of arguments for the sycl
+ * interoperability kernel.
  *
- * @param    KRef           DPCTLSyclKernelRef pointer to an OpenCL
+ * @param    KRef           DPCTLSyclKernelRef pointer to an SYCL
  *                          interoperability kernel.
- * @return   Returns the number of arguments for the OpenCL interoperability
+ * @return   Returns the number of arguments for the interoperability
  *           kernel.
  * @ingroup KernelInterface
  */
 DPCTL_API
 size_t DPCTLKernel_GetNumArgs(__dpctl_keep const DPCTLSyclKernelRef KRef);
 
 /*!
- * @brief Deletes the DPCTLSyclKernelRef after casting it to a sycl::kernel.
+ * @brief Deletes the DPCTLSyclKernelRef after casting it to a
+ * ``sycl::kernel``.
  *
- * @param    KRef           DPCTLSyclKernelRef pointer to an OpenCL
+ * @param    KRef           DPCTLSyclKernelRef pointer to an SYCL
  *                          interoperability kernel.
  * @ingroup KernelInterface
  */
 DPCTL_API
 void DPCTLKernel_Delete(__dpctl_take DPCTLSyclKernelRef KRef);
 
+/*!
+ * !brief Wrapper around
+ * `kernel::get_info<info::kernel_device_specific::work_group_size>()`.
+ *
+ * @param   KRef           DPCTLSyclKernelRef pointer to an SYCL
+ *                         interoperability kernel.
+ * @return  Returns the maximum number of work-items in a work-group
+ *          that can be used to execute a kernel on a specific device.
+ * @ingroup KernelInterface
+ */
+DPCTL_API
+size_t DPCTLKernel_GetWorkGroupSize(__dpctl_keep const DPCTLSyclKernelRef KRef);
+
+/*!
+ * !brief Wrapper around
+ * `kernel::get_info<info::kernel_device_specific::preferred_work_group_size_multiple>()`.
+ *
+ * @param   KRef           DPCTLSyclKernelRef pointer to an SYCL
+ *                         interoperability kernel.
+ * @return  Returns a value, of which work-group size is preferred to be a
```

Comment:
My reuse the docstring from the Python function corresponding to the C API function.